go/vt/vtorc/logic/topology_recovery.go

```diff
@@ -743,13 +743,17 @@ func executeCheckAndRecoverFunction(analysisEntry *inst.ReplicationAnalysis) (er
 	recoveriesCounter.Add(recoveryName, 1)
 	if err != nil {
 		message := fmt.Sprintf("Recovery failed on %s for problem %s. Error: %s", analysisEntry.AnalyzedInstanceHostname, analysisEntry.Analysis, err.Error())
-		vtopsExec.SendSlackMessage(message, vtopsSlackChannel)
+		if analysisEntry.IsPrimary {
```

Comment:
`IsPrimary` is an attribute of the vtorc analysis (i.e. diagnostic of the problem), not of the `vtorc` instance itself. So with this change, the recovery messages would only be used for primary vttablets, bout would still be sent by all `vtorc`s. Unlike classical Orchestrator, in `vtorc` there are no cluster leaders/primaries, they all operate independently of each other.